package.json

```diff
@@ -68,7 +68,7 @@
     "typescript": "3.9.7"
   },
   "peerDependencies": {
-    "typescript": "^3.4.5"
+    "typescript": ">=3.4.5"
```

Comment:
This change is appreciated, and is required!!! @quinnturner PR **will fix** this issue for me  (below) ... I am awaiting for it with great impatience:

 With **NPM: 7.0.2** →  I can force using  `--force`, or use `--legacy-peer-deps`  the install but it dose not install by itself any more   
```log
npm install                                 
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: luxcium.io@1.0.0
npm ERR! Found: typescript@4.0.3
npm ERR! node_modules/typescript
npm ERR!   typescript@"^4.0.3" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer typescript@"^3.4.5" from @nestjs/schematics@7.1.2
npm ERR! node_modules/@nestjs/schematics
npm ERR!   @nestjs/schematics@"^7.0.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/luxcium/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/luxcium/.npm/_logs/2020-10-21T05_51_57_674Z-debug.log
```

Reply:
Meanwhile I will use `yarn install` so it automatically install without the need of a manual force or work around when using `typescript@latest`...  giving only a warning as above in the PR message: `warning " > @nestjs/schematics@7.1.2" has incorrect peer dependency "typescript@^3.4.5".`